km/km_cpu_init.c

```diff
@@ -385,16 +394,22 @@ km_vcpu_t* km_vcpu_fetch_by_tid(pid_t tid)
    return NULL;
 }
 
+static inline void km_signal_vcpu_stop(km_vcpu_t* vcpu)
+{
+   sigval_t val = {.sival_int = vcpu->vcpu_id | 0x10000};
```

Comment:
Please create a symbol for 0x10000.
It helps people know that different parts of the code are referring to the same bit.